DependencyInjection/Configuration.php

```diff
@@ -84,6 +84,19 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('When true, resolving of Doctrine mapping is done automatically to use either ORM or ODM object manager')
                     ->defaultTrue()
                 ->end()
+                ->scalarNode('token_parameter_name')->defaultValue('refresh_token')->end()
+                ->arrayNode('cookie')
```

Comment:
IMO this probably needs a `canBeEnabled()` call which would result in defaulting the cookie storage to disabled.

Reasons for this:

1) The checks in the event listener right now boil down to whether or not the cookie storage settings array is empty, which if the config defaults are being set, would never be the case
2) Enabling cookie storage by default would create a rather interesting B/C break, the cookie-based approach really only works well when you're using an API behind a SPA but in more traditional REST API applications having the token be in a cookie might be problematic.